embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/spans/EmbraceSpan.kt

```diff
@@ -75,6 +76,16 @@ public interface EmbraceSpan {
         name: String
     ): Boolean = addEvent(name = name, timestampMs = null, attributes = null)
 
+    /**
+     * Add an [EmbraceSpanEvent] with the given [name] and [timestampMs]. Optionally, a set of attributes associated with the event can
+     * be passed in. Returns false if the Event was definitely not successfully added. Returns true if the validation at the Embrace
```

Comment:
I don't see how it's possible to pass attributes in

Reply:
I just added an overload - there is already one that has attributes passed in